azure-container-apps/ADVANCED.md

```diff
@@ -0,0 +1,321 @@
+# Advanced details for your 1Password SCIM Bridge on Azure Container Apps
+
+_Learn how to deploy 1Password SCIM Bridge on the [Azure Container Apps](https://azure.microsoft.com/en-us/products/container-apps/#overview) service._
+
+This guide goes into more advanced cusomtizations or configurations for the Azure Container App deployment of your 1Password SCIM Bridge. There are a few benefits to deploying 1Password SCIM Bridge on Azure Container Apps:
+
+- **Low cost:** For standard deployments, the service will host your SCIM bridge for ~$16 USD/month (as of January 2024). Container Apps pricing is variable based on activity, and you can learn more on [Microsoft's pricing page](https://azure.microsoft.com/en-us/pricing/details/container-apps/).
+- **Automatic DNS record management:** You don't need to manage a DNS record. Azure Container Apps automatically provides a unique one for your SCIM bridge domain.
+- **Automatic TLS certificate management:** Azure Container Apps automatically handles TLS certificate management on your behalf.
+- **Multiple deployment options:** The SCIM bridge can be deployed directly to Azure from the Portal using [this guide](README.md) or via the Azure Shell or command line tools in your local terminal using the [support guide](https://support.1password.com/scim-deploy-azure/). If you're using a custom deployment, cloning this repository is recommended.
+
+**Table of contents:**
+
+- [Resource recommendations](#resource-recommendations)
+- [Customize your deployment](#customize-your-deployment)
+- [Get help](#get-help)
+- [Updating your SCIM Bridge](#updating-your-scim-bridge-in-the-azure-cloud-shell)
```

Comment:
Hmm, why is this section under get help? It should be its own H2. Also the title [shouldn't use -ing](https://support.1password.com/style-guide/#-ing-words) if possible:

```suggestion
- [Update your SCIM Bridge](#update-your-scim-bridge-in-the-azure-cloud-shell)
```